Add configuration for Data Center URL (#8)
mParticle/mparticle-java-events-sdk: README.md

                .createService(EventsApi.class);
```

> By default, the Java SDK will upload to the [US1 Data Center URL](https://docs.mparticle.com/developers/server/http/#endpoint).  If the API key you're sending to exists in an mParticle Data Center that's not US1, you can add the following configuration to the SDK initialization:

Reviewer: ```suggestion
By default, the Java SDK will upload to the [US1 Data Center URL](https://docs.mparticle.com/developers/server/http/#endpoint).  If the API key you're sending to exists in an mParticle Data Center that's not US1, you must override the base url using the following configuration:
```

```java

> Retrofit.Builder mybuilder = new Retrofit.Builder().baseUrl("DATA CENTER URL").build();

Reviewer: ```suggestion
Retrofit.Builder mybuilder = new Retrofit.Builder().baseUrl("https://s2s.eu1.mparticle.com/v2/").build();
```

Reviewer: if i were a developer i'd want to know if i need to start it with https, end it with a slash, should it contain `/events` etc - so better to include a real example

Author: smart!

EventsApi api = new ApiClient("API KEY", "API-SECRET")
                .setAdapterBuilder(mybuilder)
                .createService(EventsApi.class);
```

> 

Reviewer: [See the mParticle docs site](https://docs.mparticle.com/developers/data-localization/#events-api) to find the right URL for your hosting location.

Author: @samdozor I added the piece about defaulting to US1 Data Center, which links to mP docs.  What are you suggesting with this comment?
